src/popup/components/form/FormAccountInput.vue

```diff
@@ -0,0 +1,246 @@
+<template>
+  <InputAccount
```

Comment:
As I mentioned in the `InputAccount` file: the form fields should extend the base `InputField` component.